### ISSUE TITLE
Wrong APC and missing license

### ISSUE THREAD

**Issue by restrepo** [colav-playground/advanced_user_tests#3]
In the current DOAJ database, it must be understood that the journals which do not have APC, means that the corresponding APC is 0 USD, instead of empty or `None`.

For example, for the _Journal of High Energy Physics_ with ISSN: 1029-8479'
```python
>>> import pandas as pd
>>> djf=pd.read_json('https://github.com/colav-playground/advanced_user_tests/raw/main/data/journal_batch_1.json.gz')
>>> djf[djf['bibjson'].str['eissn']=='1029-8479']['bibjson'].str['apc'].iloc[0]
{'has_apc': False,
 'url': 'http://www.springer.com/physics/particle+and+nuclear+physics/journal/13130'}
>>> djf[djf['bibjson'].str['eissn']=='1029-8479']['bibjson'].str['license'].iloc[0][0]['type']
'CC BY'
```
`'has_apc': False,` means  0 USD, with license 'CC BY'

However, for the work:

* `_id:"641b90401654767daee8531f"`
* doi: "10.1007/jhep04(2020)134"
 the  information `source -> apc`  is empty instead 0 USD, and `source -> license` is still not included (Yellow in data scheme):

```python
{'id': '6418859357aefa51300b5b46',
 'names': [{'lang': 'en', 'name': 'Journal of High Energy Physics'}],
 'external_ids': [{'source': 'eissn', 'id': '1029-8479'},
  {'source': 'scimago', 'id': '28520'},
  {'source': 'openalex', 'id': 'https://openalex.org/V187585107'}],
 'ranking': [{'to_date': 1640995199,
   'from_date': 1640995199,
   'rank': 'Q2',
   'order': 4834,
   'source': 'scimago Best Quartile'},
  {'to_date': 1640995199,
   'from_date': 1640995199,
   'rank': 238,
   'order': 4834,
   'source': 'scimago hindex'},
  {'to_date': 1640995199,
   'from_date': 1640995199,
   'rank': 0.895,
   'order': 4834,
   'source': 'scimago'},
  {'from_date': 1415635087,
   'to_date': 1415635087,
   'rank': 'C',
   'issn': '1029-8479',
   'order': None,
   'source': 'scienti'}],
 'apc': {},
 'waiver': {'has_waiver': False},
 'addresses': []}
```

**Comment by muzgash** [comment]
0 is not wrong, is by design. There is a big difference between having or not having a value. None is when the variable is not available, 0 must say that the variable is available and takes the value of 0.

**Comment by restrepo** [comment]
In the original data base of DOAJ the equivalent variable 'has_apc': is `False`. Which is neither 0 nor `None`. 

The issue is that in our data scheme  it wrongly interpreted as an empty `'apc': {}`

Please solve the issue in a such way that reflect the fact that "There are NO PUBLICATION FEES" for this of journal, as explicitly stablished in the DOAJ profile for this specific journal: [https://doaj.org/toc/1029-8479](https://doaj.org/toc/1029-8479)

**Comment by muzgash** [comment]
Fixed in version 0.0.3a0 on repository: https://github.com/colav/Kahi_plugins/tree/doaj_sources
